src/services/users/index.ts

```diff
@@ -5,6 +5,8 @@ import httpClient from '../http-client';
 import globalRoles from './globalRoles';
 import groupRoles from './groupRoles';
 import health from './health';
+import { userSettingsService } from './settings';
```

Comment:
(Stupid) Could also just be `Settings` or `SettingsService` because we're already in the users context here?